Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -186,7 +187,7 @@ - (void)sentrySessionStarted:(SentrySession *)session
 
 - (void)captureReplay
 {
-    [self.sessionReplay captureReplay];
+    //[self.sessionReplay captureReplay];
```

Comment:
Was this meant to be commented out in the final version? It stopped hybrid SDKs from sending replays.

Reply:
forgot to tag @brustolin 

cc/fyi @krystofwoldrich if you notice stuff doesn't work for you in RN